build-system/pr-check/utils.js

```diff
@@ -131,6 +136,29 @@ async function startSauceConnect(functionName) {
   execOrDie(startScCmd);
 }
 
+/**
+ * Start DNS monitor
+ * @return {!Promise}
+ */
+function startDnsMonitor_() {
+  // if (!isTravisBuild()) {
+  //   return Promise.resolve(null);
+  // }
```

Comment:
Is this necessary? What is the current behavior during local development?

Reply:
Not strictly necessary because only `pr-check/xxx.js` references to it. I put it back as an extra layer of protection in case it is needed in special circumstances. It is supposed to be turned off during local dev, since
1. You might be doing other stuffs on your computer while running tests, and it can't distinguish them from those generated from tests
2. This needs to be ran as root, which is doable but needs some special handling.

Reply:
Gotcha. In that case, we should just run this on Travis. Speaking of which, Travis VMs do background stuff too. For instance, if you're using this during a sauce labs run, there is a proxy on the Travis VM communicating with the Sauce VM.

Is it safe to say this should only be used for local unit, integration, and e2e tests?

/cc @estherkim for thoughts here.

Reply:
I haven't turned it on for sauce lab runs. If we need to, we could whitelist them. (I've whitelisted Google Cloud-specific stuff travis uses and app engine stuff infra team uses) As long as they remain constant, it's not a problem.